src/System.Runtime.CompilerServices.Unsafe/tests/Configurations.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard1.3;
+      netstandard1.3-Windows_NT;
```

Comment:
Is this just because we don't have ildasm for non-windows yet? @mellinoe If so it seems like we should have a tracking issue with a comment here.